src/scss/_type.scss

```diff
@@ -37,9 +37,10 @@
   font-family: Distinct, sans-serif;
 }
 
-@mixin underline($padding: 0, $width: 0.0625rem, $color: $pure-black) {
-  border-bottom: $width solid $color;
-  padding-bottom: $padding;
+@mixin underline($offset: 8px, $thickness: 1px, $color: $pure-black) {
```

Comment:
should the offset be a spacing token by default? I suspect so because as the text gets larger, the offset needs to increase

Reply:
![image](https://github.com/user-attachments/assets/33c3f903-3aec-45f8-aae2-8edf8b7e19d2)
![image](https://github.com/user-attachments/assets/3bc06cef-ff47-48c5-96ee-dca6b68aa5d0)
![image](https://github.com/user-attachments/assets/7b2a1aaa-e716-478d-b9dd-cab159ee3db7)

still looks good on different size